src/__tests__/getByApi.test.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React from 'react';
+import { View, Text, TextInput, Button } from 'react-native';
+import { render } from '..';
+
+const MyComponent = () => {
+  return <Text>My Component</Text>;
+};
+
+test('byTestId returns only native elements', () => {
+  const {
+    getByTestId,
+    getAllByTestId,
+    queryByTestId,
+    queryAllByTestId,
+  } = render(
+    <View>
+      <Text testID="text">Text</Text>
+      <TextInput testID="textInput" />
+      <View testID="view" />
+      <Button testID="button" title="Button" onPress={jest.fn()} />
+      <MyComponent testID="myComponent" />
+    </View>
+  );
+
+  expect(getByTestId('text')).toBeTruthy();
+  expect(getByTestId('textInput')).toBeTruthy();
+  expect(getByTestId('view')).toBeTruthy();
+  expect(getByTestId('button')).toBeTruthy();
+
+  expect(getAllByTestId('text')).toHaveLength(1);
+  expect(getAllByTestId('textInput')).toHaveLength(1);
+  expect(getAllByTestId('view')).toHaveLength(1);
+  expect(getAllByTestId('button')).toHaveLength(1);
+
+  expect(queryByTestId('myComponent')).toBeFalsy();
```

Comment:
can we add `getByTestId` that fails for completeness?

Reply:
Sure, I've replaced queryBy with getBy for negative cases.